docs/docs/sourcing-from-yaml.md

```diff
@@ -0,0 +1,232 @@
+---
+title: "YAML as a source for building a site"
```

Comment:
We should use more active titles that communicate the job to be done:
```suggestion
title: "Building a Gatsby site from YAML"
```

Reply:
Title changed